pkg/tsdb/cloud-monitoring/converter/converter.go

```diff
@@ -24,7 +24,7 @@ type Options struct {
 }
 
 func rspErr(e error) backend.DataResponse {
-	return backend.DataResponse{Error: e}
+	return backend.DataResponse{Error: e, ErrorSource: backend.ErrorSourceDownstream}
```

Comment:
This method is used at couple of places and I am wondering if it is correct to mark everything as downstream. I believe we should use `backend.IsDownstreamError(err)` to check if the error is really downstream. And then mark downstream errors with `backend.DownstreamError` if they are downstream. 

Or alternatively, if we want to mark all errors as Downstream, I would rename the function to something like `rspDownstremErr`.

Reply:
Ty for the review! I have renamed the function for clarity. AFAICT any of these errors are due to data parsing issues or other response issues so I think it's fine to mark them all as downstream 😊